### DISCUSSION TITLE
C API Support

### DISCUSSION THREAD

**Discussion by prasadakhil97** [microsoft/msquic#2057] [Q&A]
Hello, 
I am a graduate student playing around MsQUIC for one of my projects. 
I was interested in using MsQUIC C API for my project. 
While I tried to run the sample code provided, it's giving me errors during compilation itself.
Any pointers regarding this would be helpful.

**Comment by nibanks** [comment]
Please copy the details of your build errors here.

**Reply by prasadakhil97** [reply]
I was able to run the Sample program. 
I had one more query. 
If write my program, do I need to create a separate entry for that in the Makefile of /build directory?

**Comment by prasadakhil97** [comment]
Now, I am getting ListenerStart Error with a code of 0x61 when I run the Server. 
What might be the issue for this
